src/EnergyPlus/SolarShading.cc

```diff
@@ -6179,7 +6188,10 @@ namespace SolarShading {
 
 				if ( SunLitFract > 0.0 && Surface( SurfNum ).Class != SurfaceClass_TDD_Dome ) {
 
-					if ( ShadeFlag == IntShadeOn || ShadeFlag == ExtShadeOn || ShadeFlag == IntBlindOn || ShadeFlag == ExtBlindOn || ShadeFlag == BGShadeOn || ShadeFlag == BGBlindOn || ShadeFlag == ExtScreenOn ) {
+					if ( SurfaceWindow( SurfNum ).WindowModelType != WindowBSDFModel &&
+						( ShadeFlag == IntShadeOn || ShadeFlag == ExtShadeOn || ShadeFlag == IntBlindOn ||
+							ShadeFlag == ExtBlindOn || ShadeFlag == BGShadeOn || ShadeFlag == BGBlindOn ||
+							ShadeFlag == ExtScreenOn ) ) {
 						TBmAll = TBmAllShBlSc;
```

Comment:
I know the style guide mentions wrapping long lines, but we aren't doing that yet per the wiki comment.  @Myoldmopar ?

Reply:
I will unwrap them for now. I would vote if we can in future wrap them. It is hard to read them in this way.